src/pages/index.tsx

```diff
@@ -1,63 +1,15 @@
 import type { NextPage } from "next";
-import styled from "styled-components";
-import { useStoreDispatch, useStoreSelector } from "../store/hooks";
-import {
-  decrement,
-  increment,
-  clear,
-  incrementByAmount,
-  changeCounter,
-  getTestData,
-} from "../store/reducer/counterSlice";
 import HeaderLayout from "../layouts/HeaderLayout";
-import { Button, Box, Flex, Heading, Text } from "@chakra-ui/react";
-import ChangeCourtSize from "@/components/ChangeCourtSize";
+import dynamic from "next/dynamic";
+
+const BasketballCourt = dynamic(() => import("@/components/BasketballCourt"), { ssr: false });
```

Comment:
Please explain why needs lazy loading on this component?

Reply:
We treid to remove this lazy loading, but we found that it would throw an exception. The only way to fix it is to import it dynamically. Do you have any suggestions about that?
https://github.com/konvajs/react-konva/issues/588

Reply:
I think you are right. I haven't found a useful solution for ESM mode on Nextjs yet. The official document may lead to a closer solution but it not working itself.
https://nextjs.org/blog/next-11-1#es-modules-support